tests/main/null/task.yaml

```diff
@@ -0,0 +1,6 @@
+summary: Dummy test case which does nothing
```

Comment:
Couldn't we use one of the existing tests for this? The `test/main/help` variants are very simple, as soon as `snap` is available from the command line they should work.